autoparallel/_passes/split_fsdp_collectives.py

```diff
@@ -49,12 +50,19 @@ class EpilogueInput(AOTOutput):
     pass
 
 
-def split_fsdp_prefetch(g: torch.fx.Graph) -> tuple[torch.fx.Graph, torch.fx.Graph]:
-    g_ins = g.find_nodes(op="placeholder")
+def split_fsdp_prefetch(
+    gm: torch.fx.GraphModule,
+    num_params: int,
```

Comment:
If we use export with descriptors, num_params could be taken from it.  I will check it.